openhands/agenthub/codeact_agent/micro/npm.md

```diff
@@ -0,0 +1,9 @@
+---
+name: npm
+agent: CodeActAgent
+triggers:
+- npm
+---
+
+When using npm to install packages, you will not be able to use an interactive shell, and it may be hard to confirm your actions.
+As an alternative, you can pipe in the output of the unix "yes" command to confirm your actions.
```

Comment:
npm could just be called with "-y" flag, iirc

Reply:
Thanks! It can be, but that doesn't work when we use things like "create-vite" (as I briefly mentioned in the original PR).